wd_helper.au3

```diff
@@ -1155,6 +1155,85 @@ Func _WD_jQuerify($sSession, $sjQueryFile = Default, $iTimeout = Default)
 	Return SetError(__WD_Error($sFuncName, $iErr, $sParameters))
 EndFunc   ;==>_WD_jQuerify
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_FindElementFrames
+; Description ...: Check if element exist in all documents (including frames) and return their location
```

Comment:
Please review Description and suplement heplper function list here:
https://github.com/Danp2/au3WebDriver/blob/master/README.md#documentation

Reply:
The description needs work. My first crack at it is "Search the current document (including frames) and return the location of any matching elements". 

Next you'll need to define what "location" means. Perhaps an entry under "Remarks"

Reply:
`Search the current document (including frames) and return the location of any matching elements`

Any suplement/fix ... if we takes into consideration that matching elements can be found on multiple frames ? I mean list of frames location will be returned as array.

Reply:
> Any suplement/fix ... if we takes into consideration that matching elements can be found on multiple frames ? I mean list of frames location will be returned as array.

I don't understand this last comment. Isn't this already covered by `return the location of any matching elements`?

Reply:
I re-read it and all is Ok here.

Reply:
> Next you'll need to define what "location" means. Perhaps an entry under "Remarks"

What you think about my proposed remark:
`Returned location (path like 'null/2/0') can be used with _WD_FrameEnter before _WD_FindElement or _WD_WaitElement will be used.`

Reply:
I think it is too wordy / confusing. I suggest changing it to "The returned location can be used with _WD_FrameEnter to switch to the target frame."

Reply:
Please take a look: [Remarks](https://github.com/Danp2/au3WebDriver/pull/386/commits/e0eb4fac253f52affc6da250cdd83930adfc10bf)